loop/routes/rooms.js

```diff
@@ -201,25 +201,27 @@ module.exports = function (apiRouter, conf, logError, storage, auth,
       roomData.updateTime = now;
       roomData.expiresAt = now + roomData.expiresIn * tokenlib.ONE_HOUR;
       roomData.roomOwnerHmac = req.user;
+      roomData.channel = req.body.channel || "default";
```

Comment:
We need it when a new participant arrive to generate it's session_token: https://github.com/mozilla-services/loop-server/pull/278/files#diff-2fbbe66e501e7e82cd4c1868a4632212R344